### PR TITLE
fix: advise setting 'CRC32C_PURE_PYTHON' after build failure

### PR DESCRIPTION
**Pull request by tseaver**
Toward #83.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,15 +12,19 @@
...
 import setuptools.command.build_ext
-import warnings
 
 _EXTRA_DLL = "extra-dll"
 _DLL_FILENAME = "crc32c.dll"
-CRC32C_PURE_PYTHON = os.getenv("CRC32C_PURE_PYTHON") is not None
+
+# Explicit environment variable disables pure-Python fallback
+CRC32C_PURE_PYTHON_EXPLICIT = "CRC32C_PURE_PYTHON" in os.environ
+_FALSE_OPTIONS = ("0", "false", "no", None)
```
Comment (parthea) [inline setup.py:26]: Can we also add `"False", "No"` ?

File: setup.py [hunk 2]
```diff
@@ -54,14 +58,27 @@ def run(self):
...
+            packages=["google_crc32c"],
+            package_dir={"": "src"},
+            ext_modules=[module],
+            cmdclass={"build_ext": BuildExtWithDLL},
+        )
+    except SystemExit:
+        if "CRC32C_PURE_PYTHON" not in os.environ:
+            # If build / insall fails, it is likely a compilation error with
+            # the C extension:  adviser user how to enable th pure-Python
+            # build.
```
Comment (parthea) [inline setup.py:78]:
```suggestion
            # If build / install fails, it is likely a compilation error with
            # the C extension:  advise user how to enable the pure-Python
            # build.
```

**Review by parthea — APPROVED** [review]
Added 2 minor observations, otherwise LGTM.

File: setup.py (hunk 1 above)
Comment (tseaver) [reply setup.py:26]:
```suggestion
_FALSE_OPTIONS = ("0", "false", "no", "False", "No", None)
```

**Review by andrewsg — APPROVED** [review]
Thanks!

**Pull request merged**